src/Session.php

```diff
@@ -249,8 +249,8 @@ public static function setPath()
     public static function start()
     {
         if (session_status() === PHP_SESSION_NONE) {
-            ini_set('session.use_only_cookies', '1'); // Force session to use cookies
-            session_name(self::buildSessionName());
+            @ini_set('session.use_only_cookies', '1'); // Force session to use cookies
+            @session_name(self::buildSessionName());
```

Comment:
I do not really like the idea of silenting them, is this still needed in the current version of your code ?

Reply:
On each `progressChecker->save($progress), I have this kind of issue: 
![image](https://github.com/user-attachments/assets/c0f47c88-1023-4ec2-89bf-01d2d600fe80)

So far, I'm not sure how to fix it other than silencing the warnings here 😕

Reply:
It's solved: createSchema kept displaying a space character for BC with older http request handlers

Reply:
I move their initialiazation to do it at application bootstrap, see https://github.com/glpi-project/glpi/pull/18331